Documentation/security/network/encryption-ipsec.rst

```diff
@@ -21,6 +21,22 @@ Packets are not encrypted when they are destined to the same node from which
 they were sent. This behavior is intended. Encryption would provide no benefits
 in that case, given that the raw traffic can be observed on the node anyway.
 
+v1.18 Encrypted Overlay
+=========================
+Prior to v1.18 IPsec encryption was performed before tunnel encapsulation.
+From Cilium v1.18 and forward Cilium's IPsec encryption datapath will send traffic
```

Comment:
Some commas to help non-native speakers parse sentences:
"Prior to v1.18,"
"From Cilium v1.18 and forward,"